packages/env/lib/env.js

```diff
@@ -355,6 +364,35 @@ async function copyCoreFiles( fromPath, toPath ) {
 	} );
 }
 
+/**
+ * Makes the WordPress content directories (wp-content, wp-content/plugins,
+ * wp-content/themes) owned by the www-data user. This ensures that WordPress
+ * can write to these directories.
+ *
+ * This is necessary when running wp-env with `"core": null` because Docker
+ * will automatically create these directories as the root user when binding
+ * volumes during `docker-compose up`, and `docker-compose up` doesn't support
+ * the `-u` option.
+ *
+ * See https://github.com/docker-library/wordpress/issues/436.
+ *
+ * @param {string} environment The environment to check. Either 'development' or 'tests'.
```

Comment:
Minor suggestion. I've had the feedback before to document this slightly differently, I think since this was merged:
https://github.com/WordPress/gutenberg/pull/18920/files#diff-78efab576563261e860347a820aa0997R234

You can inline a bit like this or I think consider a typedef if alignment gets difficult.
```suggestion
 * @param {'development'|'tests'} environment The environment to check.
```

Reply:
Good idea! This exists in a few other places and I can spot a few other JSDoc mistakes so I'll tackle this separately.